docs/vectorizer-api-reference.md

```diff
@@ -518,10 +537,35 @@ considerations.
 
 The available functions are:
 
+- [ai.scheduling_default](#aischeduling_default): to use the plwhen you want manual control over when the vectorizer runs. Use this when you're using an external scheduling system, as is the case with self-hosted deployments.
```

Comment:
```suggestion
- [ai.scheduling_default](#aischeduling_default): to use when you want manual control over when the vectorizer runs. Use this when you're using an external scheduling system, as is the case with self-hosted deployments that use the pgai vectorizer worker.
```

I added the last part to tie it with the vectorizer worker. Feel free to remove that part, the first part it's a typo

Reply:
This was actually a copy-paste error, thanks for the catch.